locale/en/get-involved/node-meetups.md

```diff
@@ -23,10 +23,289 @@ REQUIREMENTS
 
 ## Meetups
 
+#### Argentina
+
+##### Buenos Aires
+
+- [Meetup](https://www.meetup.com/banodejs/)
+- Frequency of meetups - monthly
+- How to submit a talk? Write a message in the meetup page
+- Organizer name - Alejandro Oviedo
+- Organizer contact info - alejandro.oviedo.g@gmail.com
+
+#### Australia
+
+##### Melbourne
+
+- [Meetup](https://www.meetup.com/NodeMelbourne/)
+- Frequency of meetups - monthly
+- How to submit a talk? Contact organizers in the meetup page.
+- Organizer name -Andrey Sidorov
+- Organizer contact info -sidorares@yandex.com
```

Comment:
Some trivial nits: could you please be so kind as to update all info following the format as below :)
```suggestion
- Organizer contact info - sidorares@yandex.com
```